Sources/BraveWallet/Crypto/Stores/CryptoStore.swift

```diff
@@ -528,8 +539,18 @@ extension CryptoStore: BraveWalletKeyringServiceObserver {
     rejectAllPendingWebpageRequests()
   }
   public func keyringCreated(_ keyringId: String) {
+    Task { @MainActor [weak self] in
+      if let newCoin = WalletConstants.supportedCoinTypes.first(where: { $0.keyringId == keyringId }) {
+        self?.userAssetManager.migrateUserAssets(for: newCoin)
```

Comment:
I think we need to update `PortfolioStore` after migration? Fresh install does not show assets until tapping `Done` in Edit User Assets or opening/closing wallet.

https://github.com/brave/brave-ios/assets/5314553/7d529b69-a496-4cb9-967d-1c7124413c13